recipes/base_conanfile/all/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 cmake_minimum_required(VERSION 3.4)
 project(wrapper CXX)
 
-include(conanbuildinfo.cmake)
-conan_basic_setup(TARGETS)
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
```

Comment:
🚨  If using `TARGETS` the library `AnyAudioImported` is propagated to `Magnum` targets and linking fails because it cannot be found (it is in a different directory).

Need to understand how this library reaches the linker and what can be done to avoid it.